src/Surfnet/StepupMiddleware/CommandHandlingBundle/Identity/Service/RegistrationMailService.php

```diff
@@ -113,40 +108,43 @@ public function sendRegistrationEmailWithRas(
             $this->fallbackLocale
         );
 
+        // In TemplatedEmail email is a reserved keyword, we also use it as a parameter that can be used in the mail
+        // message, to prevent having to update all templates, and prevent a 500 error from the mailer, we perform a
+        // search and replace of the {email} parameter in the template.
+        $emailTemplate->htmlContent = str_replace(
+            '{email}',
+            '{emailAddress}',
+            $emailTemplate->htmlContent
+        );
         $parameters = [
-            'templateString'   => $emailTemplate->htmlContent,
-            'locale'           => $locale,
-            'commonName'       => $commonName,
-            'email'            => $email,
+            'templateString' => $emailTemplate->htmlContent,
+            'locale' => $locale,
+            'commonName' => $commonName,
+            'emailAddress' => $email,
             'registrationCode' => $registrationCode,
-            'expirationDate'   => $expirationDate,
-            'ras'              => $ras,
-            'selfServiceUrl'   => $this->selfServiceUrl,
+            'expirationDate' => $expirationDate,
+            'ras' => $ras,
+            'selfServiceUrl' => $this->selfServiceUrl,
         ];
 
-        $email = (new TemplatedEmail())
+        $message = new TemplatedEmail();
+        $message
             ->from(new Address($this->sender->getEmail(), $this->sender->getName()))
             ->to(new Address($email->getEmail(), $commonName->getCommonName()))
             ->subject($subject)
             ->htmlTemplate('@SurfnetStepupMiddlewareCommandHandling/SecondFactorMailService/email.html.twig')
             ->context($parameters);
-        $this->mailer->send($email);
+        $this->mailer->send($message);
     }
 
     /**
-     * @param string $locale
-     * @param string $commonName
-     * @param string $email
-     * @param string $registrationCode
-     * @param DateTime $expirationDate
-     * @param RaLocation[] $raLocations
      * @throws TransportExceptionInterface
      */
     public function sendRegistrationEmailWithRaLocations(
-        $locale,
-        $commonName,
-        $email,
-        $registrationCode,
+        string $locale,
+        string $commonName,
+        string $email,
```

Comment:
Maybe use $emailAddress iso $email makes it more readable and understandable later on.